xla/tests/onednn_matmul_test.cc

```diff
@@ -1442,6 +1442,52 @@ TEST_F(MatmulTest, WeightsPrepackAndScratch) {
   )");
 }
 
+TEST_F(MatmulTest, ConsecutiveBinaryAdd) {
+  const char* matmul_module_str = R"(
+  HloModule matmul.test.f32
+    region_0.22 {
+    Arg_0.23 = f32[] parameter(0)
+    Arg_1.24 = f32[] parameter(1)
+  ROOT add.25 = f32[] add(Arg_0.23, Arg_1.24)
+  }
+
+  region_1.29 {
+    Arg_0.30 = f32[] parameter(0)
+    Arg_1.31 = f32[] parameter(1)
+    ROOT add.32 = f32[] add(Arg_0.30, Arg_1.31)
+  }
+
+  ENTRY main {
+    constant.2 = f32[] constant(1e-06)
+    broadcast.3 = f32[1000000] broadcast(constant.2), dimensions={}
```

Comment:
I don't think the size needs to be this big to reproduce the failure. Would 10 work?

Reply:
The issue is not reproducible with a smaller size.